### PR TITLE
Improved a comment in code example for service definitions doc

### PR DESCRIPTION
**Pull request by stollr**
The improved sentence in the code comment misses a word, making the comment less reasonable.

### CODE REVIEW

**Review by javiereguiluz — CHANGES_REQUESTED** [review]
I think the current text is correct, although it could be improved.

The argument of `findDefinition()` can be an ID or alias ... so you get the definition with the "app.user_config_manager" ID or alias.

How could we reword this to make it easier to understand?

**Comment by naitsirch** [comment]
> The argument of `findDefinition()` can be an ID or alias

Yes, that's why I changed the sentence to *"get the definition [...] **by** ID or alias"*

I have just taken the sentence from the doc-comment of the `findDefinition()` method. See [here](http://api.symfony.com/3.3/Symfony/Component/DependencyInjection/ContainerBuilder.html#method_findDefinition)

**Comment by javiereguiluz** [comment]
I see ... but I think the new sentence is missing something. Now it reads: `... with the XXXX by ID or alias`. There's something missing after `the XXX` or the `the` must be removed. Maybe it's more clear like this:

```
// get the definition whose ID or alias is "app.user_config_manager"
```

**Comment by naitsirch** [comment]
Oh dear, after reading the sentence once again I realize that I have got it totally wrong. Sorry for the noise, I think it is okay, like it is. I'll close this PR.
